app/src/main/java/com/raghav/spacedawn/utils/AlarmBroadCastReciever.kt

```diff
@@ -34,27 +33,25 @@ class AlarmBroadCastReciever : BroadcastReceiver() {
         }
 
         val notification = NotificationCompat.Builder(p0!!, CHANNEL_ID)
-            .setContentTitle("Reminder!")
-            .setContentText("Reminder to see the Rocket Launch set via Space Dawn")
+            .setContentTitle(Constants.REMINDER)
```

Comment:
bring this with R.strings please

Reply:
I pushed the code after making the prescribed change but will have to see why it's still showing the old code

Reply:
resolved